test/tools/ossfuzz/strictasm_diff_ossfuzz.cpp

```diff
@@ -84,7 +84,8 @@ extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
 	yulFuzzerUtil::TerminationReason termReason = yulFuzzerUtil::interpret(
 		os1,
 		stack.parserResult()->code,
-		EVMDialect::strictAssemblyForEVMObjects(langutil::EVMVersion())
+		EVMDialect::strictAssemblyForEVMObjects(langutil::EVMVersion()),
+		true
```

Comment:
Disable memory tracing in the fuzzer harness since it will lead to false positives like

```
{
   mstore(0, 1) // will be removed by RSE but if memory tracing is enabled, a trace MSTORE(0,1) and memory contents at address zero will appear in the unoptimized run but not in the optimized run
}

Reply:
Shouldn't this be a comment in the code instead? :)

Reply:
Added comment.